packages/flutter/test/material/data_table_test.dart

```diff
@@ -575,8 +575,10 @@ void main() {
   testWidgets('DataTable custom horizontal padding - checkbox', (WidgetTester tester) async {
     const double _defaultHorizontalMargin = 24.0;
     const double _defaultColumnSpacing = 56.0;
+    const double _defaultCheckboxHorizontalMargin = 24.0;
     const double _customHorizontalMargin = 10.0;
     const double _customColumnSpacing = 15.0;
+    const double _customCheckboxHorizontalMargin = 15.0;
```

Comment:
I would prefer if you add a new test, and leave this test as it is, here that uses the `_customCheckboxHorizontalMargin` so that we can confirm that the old behavior and the new behavior works.

Reply:
@perclasson made the necessary changes :+1: